lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -22,6 +22,8 @@
 from xmodule.modulestore.tests.factories import CourseFactory
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 
+from edx_notifications.stores.sql.models import SQLNotificationMessage
```

Comment:
@muhhshoaib I'd really like us to avoid importing the store provider directly in LMS tests. Is there any reason you can use the library methods in edx_notifications.api.consumer? If you need more library endpoints, let me know.